tt_llk_wormhole_b0/common/inc/sfpu/ckernel_sfpu_exp.h

```diff
@@ -268,135 +192,75 @@ constexpr auto bits = [](float x) constexpr { return __builtin_bit_cast(std::uin
 constexpr auto lo16 = [](float x) constexpr { return static_cast<std::uint16_t>(bits(x) & 0xFFFFu); };
 constexpr auto hi16 = [](float x) constexpr { return static_cast<std::uint16_t>(bits(x) >> 16); };
 
+inline void confige_faast_exp_addr_mod()
+{
+    addr_mod_t {
+        .srca = {.incr = 0},
+        .srcb = {.incr = 0},
+        .dest = {.incr = 2},
+    }
+        .set(ADDR_MOD_7);
+}
+
 template <bool APPROXIMATION_MODE, bool FAST_APPROX, uint32_t scale /* 1.0f in FP32 */>
 inline void _init_exponential_()
 {
     if constexpr (FAST_APPROX && APPROXIMATION_MODE)
     {
-        // Algorithm is adapted from:
-        //      A Fast, Compact Approximation of the Exponential Function
-        //      Nicol N. Schraudolph
-        //      IDSIA, Lugano, Switzerland
-
-        // First, set up constant values which are needed for the computation
-        //      We will first sanitize the input values (y) to be in the range that won't cause underflow, which for our hardware means we need to limit
-        //      negative values to be greater than or equal to -88.5 The computation that is needed is (A * y) + (B - C) , where A = (2^8)/ln(2) , B = 127 *
-        //      (2^8) , C = Adjustment parameter of roughly 11.2 to minimize error
-        //          - NOTE: we would like to be able to use 2^23 instead of 2^8 and compute a 32-bit quantity, but our hardware only supports rounding FP32 into
-        //          a 16-bit integer, so we use 2^8 and then shift left by 15 bits after rounding
-        //      So we will set up the following constants:
-        //          LREG[14] =       =    -88.5               = 0xc2b10000
-        //          LREG[12] = A     =    369.329925537109375 = 0x43b8aa3b
-        //          LREG[13] = (B-C) =  32500.818359375       = 0x46fde9a3
-
-        constexpr float LN2_RECIP = 1.4426950408889634f;
-        constexpr float A         = 256.0f * LN2_RECIP;
-        constexpr float B_minus_C = 32500.818359375f;
-        constexpr float THRESHOLD = -88.5f;
-
         constexpr float scale_fp32 = __builtin_bit_cast(float, scale);
 
-        constexpr float A_scaled         = A * scale_fp32;
-        constexpr float THRESHOLD_scaled = THRESHOLD / scale_fp32;
+        constexpr float A_scaled         = 369.329925537109375 * scale_fp32;
+        constexpr float THRESHOLD_scaled = -86.6 / scale_fp32;
+        constexpr float B_MINUS_C        = (32500.818359375 + 12582912 * 2);
+
+        confige_faast_exp_addr_mod();
 
         TTI_SFPLOADI(0, 0xA, lo16(THRESHOLD_scaled));
         TTI_SFPLOADI(0, 0x8, hi16(THRESHOLD_scaled));
-        TTI_SFPCONFIG(0, 14, 0); // SFPCONFIG Dest 14 = LREG[14] =            -88.5               = 0xc2b10000
+        TTI_SFPCONFIG(0, 14, 0); // SFPCONFIG Dest 14 = LREG[14]         =    -86.6               = 0xC2AD_3333
+
+        TTI_SFPLOADI(0, 0xA, lo16(B_MINUS_C));
+        TTI_SFPLOADI(0, 0x8, hi16(B_MINUS_C));
```

Comment:
```suggestion
        TTI_SFPLOADI(ckernel::p_sfpu::LREG0, sfpi::SFPLOADI_MOD0_LOWER, lo16(B_MINUS_C));
        TTI_SFPLOADI(ckernel::p_sfpu::LREG0, sfpi::SFPLOADI_MOD0_UPPER, hi16(B_MINUS_C));
```
It'd be nice to replace the magic numbers with some constants. It's much easier to read the code later.